core/modules/node/node.module

```diff
@@ -1201,14 +1201,17 @@ function node_permission() {
       'title' => t('Bypass content access control'),
       'description' => t('View, edit and delete all content regardless of permission restrictions.'),
       'restrict access' => TRUE,
+      'warning' => t('Allows people full access to content, regardless of other node permissions defined here.'),
```

Comment:
```suggestion
      'warning' => t('Allows people full access to content, regardless of other content permissions defined here.'),
```

Reply:
I purposefully used the word 'node' here since that is what this section of permissions is called:

![image](https://user-images.githubusercontent.com/2385329/158910167-78b8267b-cf79-4429-809a-d87a89a2a4f2.png)